tests/helpers/bn.ts

```diff
@@ -0,0 +1,9 @@
+import { BigNumber, BigNumberish } from "ethers";
+
+const APPROX_EQUAL_THRESHOLD = 10;
+
+export const approxEqual = (value: BigNumberish, equality: BigNumberish) => {
```

Comment:
I would rather name the params `a` & `b` (`ethers-utils`'s convention) or `value` & `target`

Reply:
not blocking though